### DISCUSSION TITLE
Retrieve data from related table

### DISCUSSION THREAD

**Discussion by chrismb23** [Budibase/budibase#8426] [Help]
I'm having trouble wrapping my mind around the following problem:

2 Budibase DB's: Articles and Orders

Articles
- Article Name
- Article Price

Orders
- Order ID
- Article Name (relationship with Articles > Article Name table)

Functionality I'm looking for is that on the Create New Order form the user selects the Article Name from a drop down and then the price is shown on the form.

Any best practice to accomplish this?

**Comment by melohagan** [comment]
Hey @chrismb23 

Does this help? https://docs.budibase.com/docs/filter-by-relationships

In this case you would replace the Card Block with a form of type *Update*.
